todo-cli.cabal

```diff
@@ -0,0 +1,54 @@
+-- This file has been generated from package.yaml by hpack version 0.20.0.
+--
+-- see: https://github.com/sol/hpack
+--
+-- hash: 592e31a8bba4a31e2540e7aea82b3da47430f4c1a8a18f898de13cd3be53b9a9
```

Comment:
Okay, I assume you don't have `package.yaml` anymore in your project :) So let's prettify this `.cabal` file. And start with removing this comment and empty line before `name`.